packages/dappmanager/src/modules/installer/getInstallerPackageData.ts

```diff
@@ -74,6 +90,7 @@ function getInstallerPackageData(
     compose: compose.output(),
     // User settings to be applied by the installer
     fileUploads: userSettings?.fileUploads,
-    dockerTimeout
+    dockerTimeout,
+    running
```

Comment:
We should know specifically which containers are running and which are not. If the package DMS has a prometheus and grafana services, where only prometheus is stopped; this code should
- Not start prometheus
- Start grafana